go/common/sgx/pcs/tcb.go

```diff
@@ -742,3 +742,25 @@ type EnclaveTCBLevel struct {
 	Status      TCBStatus `json:"tcbStatus"`
 	AdvisoryIDs []string  `json:"advisoryIDs"`
 }
+
+// SignedTCBEvaluationDataNumbers is the signed TCB evaluation data numbers response body.
+type SignedTCBEvaluationDataNumbers struct {
+	Numbers   TCBEvaluationDataNumbers `json:"tcbEvaluationDataNumbers"`
+	Signature string                   `json:"signature"`
```

Comment:
Should this signature be verified?

Reply:
I don't see a need for it in this case, this is being done in an untrusted environment (host node) and we just need a list of numbers to request from the same service. Verifying the signature requires a bunch of extra logic for no clear benefit.